src/commands/BeautifyCommand.ts

```diff
@@ -112,6 +113,32 @@ export class BeautifyCommand extends BaseCommand {
       });
     });
   }
+
+  private beautifySuccess({
```

Comment:
I do not think `beautifySuccess` describes *what* it does, only *when* to call it.

Something like `writeToFileOrStdout` would be better.

Reply:
@Glavin001 changed it.